pkg/pkg.mk

```diff
@@ -21,8 +21,8 @@ $(PKG_BUILDDIR)/.git-downloaded:
 
 clean::
 	@test -d $(PKG_BUILDDIR) && { \
-		git -C $(PKG_BUILDDIR) clean -f ; \
-		git -C $(PKG_BUILDDIR) checkout "$(PKG_VERSION)"; \
+		cd $(PKG_BUILDDIR) && git clean -f ; \
+		cd $(PKG_BUILDDIR) && git checkout "$(PKG_VERSION)"; \
```

Comment:
this, however, is a single line and will need to be handled. It can be sub shelled by using parentheses around the lines